spdx-utils/src/main/resources/declared-license-mapping.yml

```diff
@@ -345,6 +345,7 @@
 "Open Software License 3.0 (OSL-3.0)": OSL-3.0
 "Open Software License v. 3.0": OSL-3.0
 "Oracle Free Use Terms and Conditions (FUTC)": LicenseRef-ort-oracle-futc
+"OSI Approved": NONE
```

Comment:
I believe it'S the wrong line. Have you used the sorting of the IntelliJ IDE as stated in the file header?

Reply:
Yes, it was the wrong line. Updated now.